index.html

```diff
@@ -89,18 +82,18 @@ <h2>Workshop</h2>
         </div>
       </section>
         <section class="singers">
-            <h2 class="singerTitle">singers</h2>
+            <h2 class="singertitle">singers</h2>
               <div class="red-line"></div>
-            <div class="singersCards-preview"></div>
-            <div class="singersCards">
+            <div class="singerscards-preview"></div>
+            <div class="singerscards">
             </div>
```

Comment:
- [ ] Nice job generating the singers from JavaScript and displaying them on the page 👍 but you'll notice that the images in your design do not have the same height and width. It would be great if you could make all the images to have the same size for uniformity and also to follow the design.
- [ ] Also you will notice that each image has a decoration behind it in the design. It would be good to also add this feature to your images to improve the user interface 💯 
- Your design:
![Capture4](https://user-images.githubusercontent.com/85113567/153715591-6672da60-21f6-43b1-b7c9-436b5058ff7e.PNG)
- Required design:
![Capture10](https://user-images.githubusercontent.com/85113567/153715602-17ec6146-ac02-4b6e-882f-4c281eca44f5.PNG)

Reply:
This issue has not been resolved